site/_posts/2017-02-10-love-is-in-the-PRs.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Last Week in ManageIQ: Love is in the PRs"
+author: jntullo
+date: 2017-02-14 14:08:38 -05:00
```

Comment:
@jntullo Did you generate this post with @imtayadeway 's generator? We might want to set this to midnight to avoid a problem we've had in the past where the server thinks the post should be published in the future.